src/utils/ar_usd_oracle.ts

```diff
@@ -0,0 +1,65 @@
+import type { FiatOracle } from './fiat_oracle';
+// import CoinGecko from 'coingecko-api';
+import type { CoinGeckoPriceRequestParams, CoinGeckoPriceResponseData, CoinGeckoResponse } from './coingecko_types';
+
+const AR_COIN_ID = 'arweave';
+const USD_COIN_ID = 'usd';
+const pollingIntervalTime = 1000 * 60 * 15; // 15min expressed in in ms
```

Comment:
We might want to dial this way down for testing and then have a followup task ticket to use the real value.

Reply:
Now called cacheLifespan; that value is taken on the constructor as read-only.

Reply:
Related changes: 1986ea8139a163d1b1d73ca98b334e3405ebab99, dd0f3413271250ee23726a2bba2e0bd4056be732